### PR TITLE
SPLICE-1285 

### PR DESCRIPTION
**Pull request by jfilali**
Partially fixed the issue when the table wasn't pined to begin with. This will help to understand the underlying issue that causes the problem by removing one case bug.

### CODE REVIEW

File: db-tools-i18n/src/main/resources/com/splicemachine/db/loc/messages.xml [hunk 1]
```diff
@@ -9021,6 +9019,11 @@ Shutting down instance {0} on database directory {1} with class loader {2} </tex
                <arg>column</arg>
            </msg>
 
+           <msg>
+               <name>EXT28</name>
+               <text> Table '{0}' is not pined</text>
```
Comment (dgomezferro) [inline db-tools-i18n/src/main/resources/com/splicemachine/db/loc/messages.xml]: Typo: pinned

Comment (jfilali) [inline db-tools-i18n/src/main/resources/com/splicemachine/db/loc/messages.xml]: thanks

**Comment by dgomezferro** [comment]
@jfilali Is this still up for merging? If so please fix the conflicts, otherwise close it. Thanks!

**Comment by jfilali** [comment]
@dgomezferro  it is still up to merge, please merge in priority because it has been the 4th time I resolve conflicts

**Comment by dgomezferro** [comment]
@jfilali Yes, I'll do, ping me when the conflict is solved, thanks!

**Comment by jfilali** [comment]
@dgomezferro  Thank you Daniel

**Comment by jfilali** [comment]
@dgomezferro  done

**Comment by jfilali** [comment]
@dgomezferro  I have the 2.5 version coming in 2 min

**Pull request merged**